tools/bridge/bridge/event_fetcher.py

```diff
@@ -50,11 +31,11 @@ def __init__(
             )
 
         self.logger = logging.getLogger(
-            f"bridge.event_fetcher.{contract_address}.{event_signature}"
+            f"bridge.event_fetcher.{contract.address}.{event_name}"
```

Comment:
```suggestion
            f"bridge.event_fetcher.{contract.address.hex()}.{event_name}"
```

Reply:
Thanks for this suggestion. Good point. :laughing:

Reply:
:heavy_check_mark:

Reply:
This is of type `str` and already in hex format.

Reply:
Update:
This only holds true for the module tests. The configuration module converts address options to `bytes`. I need to find a solution for both.